content/foundations/color/base-scales.mdx

```diff
@@ -3,6 +3,73 @@ title: Base scales
 description:
   Primer base color scales makeup the foundation for all UI color considerations.
 ---
+## Neutral color scales
+
+<img
+  width="960"
+  alt="Two neutral scales stacked on top of each other labeled from 0 to 11. The top scale is the light scale, which starts with white and ends with black. The bottom scale is the dark scale, which starts with black and ends with white."
+  src="https://github.com/user-attachments/assets/3081e29d-6581-401b-91da-c014eee9cf66"
+/>
+
+Primer's neutral scales offer white, black, and shades of gray between 0 and 11. There are two versions of the neutral scale: light, and dark. 
+
+The light and dark scales are inverted respectively, with the light scale starting with white and the dark scale starting with black. By inverting the scales, light and dark themes are able to share many of the same functional color tokens without custom overrides.
+
+<img
+  width="960"
+  alt="A split view of how muted text colors reference the same base scale color in both light and dark themes."
+  src="https://github.com/user-attachments/assets/160e6384-3c33-4729-a6cf-96adce84f127"
+/>
+
+### Background
+
+The first five steps of the neutral scales are typically used for background colors. The two most commonly used background colors are `bgColor-default` and `bgColor-muted`. All contrast values for text and borders are calculated against `bgColor-muted` to ensure proper contrast ratios for both `muted` and `default`.
+
+<img
+  width="960"
+  alt="Two neutral scales stacked on top of each other labelled with background, border, and text to show how the scales are used for different UI elements. In this graphic the background portion is highlighted."
+  src="https://github.com/user-attachments/assets/1a87a17d-ce23-4c1d-8fa0-ae7b2d70aaf2"
```

Comment:
When I first looked at these graphics the purple scrim threw me off. I thought maybe that was the section I was supposed to be focusing on, or it was a demo of what these colors look like near the light purple. Maybe switch to white with more opacity, use a background blur, or just bump up the opacity of the purple?

This is definitely not a blocker and is probably just my personal misunderstanding when looking at it for the first time before reading the text and giving it some thought.

Reply:
Thank you! I'm not in love with it either. I'll keep playing 👀